p2panda-discovery/README.md

```diff
@@ -1 +1,38 @@
-Awesome documentation goes here.
+<h1 align="center">p2panda-discovery</h1>
+
+<div align="center">
+  <img src="https://raw.githubusercontent.com/p2panda/.github/main/assets/panda-left.gif" width="auto" height="30px">
+  <strong>Peer discovery traits and services</strong>
+  <img src="https://raw.githubusercontent.com/p2panda/.github/main/assets/panda-right.gif" width="auto" height="30px">
+</div>
+
+<div align="center">
+  <h3>
+    <a href="https://p2panda.org">
+      Website
+    </a>
+  </h3>
+</div>
+
+This crate is used to share address information about peers on a network. It provides a generic
+interface for the definition of a discovery service, as well as an interface for organising a
+collection of such services. An mDNS discovery implementation is provided for peer discovery over
+local networks.
+
+## Features
+
+* `Discovey`, `DiscoveryMap` and `DiscoveryEvent`
+* `LocalDiscovery` (mDNS implementation)
+
+## License
```

Comment:
There's also the option to condense the "Contributions" section into "License"; it seems that both ways are often used in Rust projects.

Reply:
Do you have an example?

Reply:
I also see an "at your option." added at the end of the "License" section.

Reply:
Here's an example from serde:

https://github.com/serde-rs/serde/tree/master#license

And from iroh:

https://github.com/n0-computer/iroh#license

Reply:
Nice, the combined version is nice, under "Contributions" I'd also expect more "community" related things like, CoC, "How to contribute" etc.

Reply:
Cool, I'll update accordingly.